extensions/binding/org.eclipse.smarthome.binding.fsinternetradio.test/src/test/java/org/eclipse/smarthome/binding/fsinternetradio/test/RadioServiceMock.java

```diff
@@ -0,0 +1,219 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.fsinternetradio.test;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.eclipse.jetty.http.HttpStatus;
+import org.eclipse.smarthome.binding.fsinternetradio.internal.radio.FrontierSiliconRadioConstants;
+
+/**
+ * Radio service mock.
+ *
+ * @author Markus Rathgeb - Migrated from Groovy to pure Java, made more robust
+ *
+ */
+public class RadioServiceMock extends HttpServlet {
```

Comment:
Maybe `RadioServiceDummy` in order to better distinguish it from Mockito-Mocks? Technically the name is correct, of course...